docs/source/sci_api_tutorial/preparation_protonate_stru.rst

```diff
@@ -0,0 +1,178 @@
+==============================================
+ Preparation: Protonate Structure
+==============================================
+
+Briefs
+==============================================
+
+This science API, named ``enzy_htp.preparation.protonate.protonate_stru``,
+solves the protein protonation problem that add missing H atoms to the supplied
+``enzy_htp.structure.Structure`` class instance (hereafter referred to as ``Structure`` instance).
+Protonation states are determined for residues with multiple ones.
+
+Input/Output
+==============================================
+
+**input**: A ``Structure`` instance (no matter it's a protein, polypeptite, or ligand).
+
+.. dropdown:: :fa:`eye,mr-1` How to obtain ``Structure`` instance?
+
+    Structure can be obtained by 
+    
+    1. parsing from a file using one of the StructureParser:
+
+    - `PDBParser <xxx>`_
+    - `PrmtopParser <xxx>`_
+
+    2. OR using the output of `Remove Solvent <preparation_remove_solvent.html>`_ or `Remove Hydrogens <preparation_remove_solvent.html>`_. (Commonly used here)
```

Comment:
mention missing loops are not acceptable

Reply:
A new line is added in dropdown box.